packages/openapi-react-query/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "openapi-react-query",
+  "description": "Fast, type-safe @tanstack/react-query client to work with your OpenAPI schema.",
+  "version": "0.0.0",
+  "author": {
+    "name": "Martin Paucot",
+    "email": "contact@martin-paucot.fr"
+  },
+  "license": "MIT",
```

Comment:
Be sure to add a `LICENSE` (no extension) file to the root of this package as well! You can find MIT license templates anywhere. Just put your name in. Even though it’s “Open Source,” you still wrote it and retain full rights to it—even when you’ve placed it in a repo like this. And you deserve to be credited as the copyright holder that is **graciously allowing** others to use your work, and you deserve credit for it 🙂

Reply:
Done!